boards/arm/black_f407ve/black_f407ve.dts

```diff
@@ -10,7 +10,7 @@
 
 / {
 	model = "black_f407ve board";
-	compatible = "st,stm32f407";
+	compatible = "unknown,black-stm32f407";
```

Comment:
I think you should just leave out the vendor prefix, then - don't introduce an `unknown` prefix.

Reply:
ok, but then, how should it be taken into account in https://github.com/zephyrproject-rtos/zephyr/pull/59642 ?

Reply:
I don't know, but adding an `unknown` vendor prefix is in violation with our binding rules: https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-vendor-prefixes

Reply:
> If your device is not manufactured by a specific hardware vendor, do not invent a vendor prefix. Vendor prefixes are not mandatory parts of compatible properties, and compatibles should not include them unless they refer to an actual vendor. There are some exceptions to this rule, but the practice is strongly discouraged.

ok, clear. I was not aware, I'll update.
Then, @nashif should take it into account in #56642, as I guess a vendor is always expected.

Reply:
@henrikbrixandersen Fixed.
@nashif Please make sure you're considering the case where no vendor name is present in #59642

Reply:
Looks good, thanks. Please update the PR description as well.